### DISCUSSION TITLE
How small can be react-native-windows

### DISCUSSION THREAD

**Discussion by minesunny** [microsoft/react-native-windows#14011] [Q&A] [answered]
I'm currently learning React Native Windows development. I want to know how small a simple demo can be after packaging.

**Comment by chrisglein** [comment, marked as answer]
There are no published numbers on the app size, and it does fluxuate between versions. But there's the RN dll, Hermes (the JS engine), the bundle... there's quite a few MBs of content there. 

Do you have app size requirements you're looking at?
